### PR TITLE
soapyremote 0.5.2 (new formula)

### PR DESCRIPTION
**Pull request by ask0n**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/s/soapyremote.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  license "BSL-1.0"
+
+  depends_on "cmake" => :build
+  depends_on "soapysdr"
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+    end
```
Comment (chenrui333) [inline Formula/s/soapyremote.rb:15]:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
```

File: Formula/s/soapyremote.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  end
+
+  test do
+    assert_match "Checking driver 'remote'... PRESENT",
+                 shell_output("#{Formula["soapysdr"].bin}/SoapySDRUtil --check=remote")
```
Comment (chenrui333) [inline Formula/s/soapyremote.rb:20]:
```suggestion
    output = shell_output("#{bin}/SoapySDRUtil --check=remote")
    assert_match "Checking driver 'remote'... PRESENT", output
```

**Comment by ask0n** [comment]
@chenrui333 this component has the same logic as https://github.com/Homebrew/homebrew-core/blob/master/Formula/s/soapyrtlsdr.rb with proposed changes CI fails...

**Comment by chenrui333** [comment]
I see, you are referencing the other formula for your formula test?

**Comment by ant-on-stellar** [comment]
> I see, you are referencing the other formula for your formula test?

No, for test original binary is used just with different parameters.
This formula and [soapyrtlsdr](https://github.com/Homebrew/homebrew-core/blob/master/Formula/s/soapyrtlsdr.rb) is just an extension driver for [soapysdr](https://github.com/Homebrew/homebrew-core/blob/master/Formula/s/soapysdr.rb)

**Comment by ask0n** [comment]
> I see, you are referencing the other formula for your formula test?

No, for test original binary is used just with different parameters.
This formula and [soapyrtlsdr](https://github.com/Homebrew/homebrew-core/blob/master/Formula/s/soapyrtlsdr.rb) is just an extension driver for [soapysdr](https://github.com/Homebrew/homebrew-core/blob/master/Formula/s/soapysdr.rb)

**Comment by chenrui333** [comment]
can you also squash the commits and add this formula into the autobump.txt? Thanks!

**Review by chenrui333 — APPROVED** [review]
lgtm

**Review by carlocab — APPROVED** [review]

**Pull request merged**